stories/avatar-group/avatar-group.stories.js

```diff
@@ -52,7 +52,7 @@ export const individualType = () => `<div class="fd-avatar-group fd-avatar-group
     <div class="fd-popover">
         <div class="fd-popover__control">
             <div class="fd-avatar-group__item" tabindex="0" aria-haspopup="true" aria-expanded="false" onclick="onPopoverClick('popover_avatar_5z28edb');">
-                <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--border" aria-label="Wendy Wallace">WW</span>
+                <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--border" role="img" alt="Wendy Wallace" title="Wendy Wallace">WW</span>
```

Comment:
I think this should be documented as a breaking change @InnaAtanasova

Reply:
@droshev if you think that it should be documented, then it should be

Reply:
@droshev it was documented on the [wiki page](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes#avatar-group--changed-structure-of-uses-avatar-component-2261)
I've added it to the PR description, thanks